Show link to annotation tool in the series LTI tool

wsmirnow opened the pull request:
The annotate option enable the button to the annotation tool for each episode of the series. Button shown in the screenshot (left of download button).

![image](https://user-images.githubusercontent.com/26736/130362312-7f0fd144-42c3-4d43-be0f-a1d5c84558f9.png)


### Your pull request should…

* [ ] have a concise title
* [ ] [close an accompanying issue](https://help.github.com/en/articles/closing-issues-using-keywords) if one exists
* [ ] [be against the correct branch](https://docs.opencast.org/develop/developer/development-process#acceptance-criteria-for-patches-in-different-versions)
* [ ] include migration scripts and documentation, if appropriate
* [ ] pass automated tests
* [ ] have a clean commit history
* [ ] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)

lkiesow commented on modules/lti/src/components/Series.tsx:
```diff
@@ -79,6 +81,10 @@ const SeriesEpisode: React.StatelessComponent<EpisodeProps> = ({ episode, delete
                     <button onClick={(e) => { editCallback(episode.id); e.stopPropagation(); }}>
                         <FontAwesomeIcon icon={faEdit} />
                     </button>}
+                {annotateCallback !== undefined &&
+                                    <button onClick={(e) => { annotateCallback(episode.id); e.stopPropagation(); }}>
+                                        <FontAwesomeIcon icon={faComments} />
+                                    </button>}
```
```suggestion
                {annotateCallback !== undefined &&
                    <button onClick={(e) => { annotateCallback(episode.id); e.stopPropagation(); }}>
                        <FontAwesomeIcon icon={faComments} />
                    </button>}
```

wsmirnow commented:
I would change the behavior to open the annotation tool in a separate browser tab. WiP.

wsmirnow committed:
Show link to annotation tool in the series LTI tool

The annotate option enable the button to the annotation tool for each episode of the series.

wsmirnow commented:
Patch rebased to current 10.x branch. Changed behavior to open annotation tool in full window.

Arnei reviewed (changes requested):
In addition to adding a note to the docs, you probably should also add a note to `lti/src/components/Welcome.tsx`. And maybe mention that the annotation tool needs to be installed for the button to actually work. Otherwise this looks good.

wsmirnow committed:
LTI series tool documentation updated

wsmirnow commented:
Thank you @Arnei for the info. I've updated the requested documentation. Please go ahead with the review.
